scripts/config/pages.js

```diff
@@ -23,7 +23,8 @@ module.exports = [
     ['new_account/welcome',                     'app/new_account/welcome_page',                'default',    'Welcome to Binary.com'],
 
     ['resources',                               'app/resources/index',                         'default',    'Resources'],
-    ['resources/asset_indexws',                 'app/resources/asset_index',                   'full_width', 'Asset Index'],
+    ['resources/asset_indexws',                'app/resources/asset_index',                   'full_width', 'Asset Index'],
+    ['resources/trading_duration',              'app/resources/trading_duration',              'full_width', 'Trading Duration'],
```

Comment:
is it trading duration or trading durations?

Reply:
it's trading durations for plural 👍